### ISSUE TITLE
wrong file in the repository?

### ISSUE THREAD

**Issue by luteknarozniak** [greerso/smOS-unsupported-miners#1]
tried swapping some unused miner with an ccminer/phi-1.0 mod from your repository, but ccminer/alexis-1.0 is being loaded instead.

**Comment by greerso** [comment]
I see the issue, looking into it now.

**Comment by greerso** [comment]
I believe the output from --version is a bug in ccminer/phi-1.0 that incorrectly states ccminer valexis-1.0 because it was forked from that project.  HiveOS shows the same incorrect--version information for the phi-1.0 miner.  the PHI1612 algorithm is included in this miner.

**Comment by luteknarozniak** [comment]
Thank you for such a quick reply - appreciate that.
That miner's been crashing like crazy, heck it can't even start hashing away. Tried different settings, with and w/o the intensity parameter, OC and stock. Nothing worked, so I thought it must be a different miner under ccminer/phi-1.0 name.
![image](https://user-images.githubusercontent.com/30240611/39008923-57338992-43c7-11e8-826c-fc3f0f66d447.png)

**Comment by greerso** [comment]
You can set the intensity in smOS by using `-i 18` or something else in the rig group on smOS

I have posted an issue regarding incorrect version information [216k155/ccminer-phi-anxmod](https://github.com/216k155/ccminer-phi-anxmod/issues/17)

I think there may be an issue with Cuda9.  I'll upload a Cuda8 version.

**Comment by luteknarozniak** [comment]
Still crashing, even faster:

![image](https://user-images.githubusercontent.com/30240611/39010896-30782924-43cd-11e8-9f32-fb921b929f31.png)

**Comment by greerso** [comment]
I have added the CUDA8 version from this thread https://github.com/216k155/ccminer-phi-anxmod/issues/15

**Comment by luteknarozniak** [comment]
Are we supposed to swap only particular SMOS miners with those new ones? e.g. ccminer-tpruvot-2.1 with ccminer-klaust-8.21 to ensure all the requirements (CUDA and other libraries) are met?

**Comment by greerso** [comment]
It shouldn't matter which you replace.  You just have to be sure to use appropriate parameters in the miner group.  What does your config look like?  I'll try it on my rig.

**Comment by luteknarozniak** [comment]
-a phi -o stratum+tcp://us1.bsod.pw:6667 -u _wallet_address_here_ -p c=LUX

**Comment by luteknarozniak** [comment]
and I think it actually does matter which one are you trying to replace. Tried swapping dstm-v0.5.6
with that phi mod and was getting errors regarding option -p being 'ambigious'. 
Just now tried to replace one of the sgminers and I'm getting 'invalid argument -a' error.

**Comment by greerso** [comment]
Sounds like something else is up.  Those options would not be invalid if the miner was properly replaced.

**Comment by luteknarozniak** [comment]
You're right, it sure does like something else is amiss. Script executes with no errors though.
Have you tried running that mod on your rig?

**Comment by luteknarozniak** [comment]
I see. 
Thank you for working with me on that issue. I appreciate that. I really do, but I don't want you to spend your afternoon chasing SMOS ghosts :)
I'll switch back to ccminer-tpruvot-2.2.5. Let's leave it up to tytanick to update/enhance miners list.

**Comment by greerso** [comment]
cuda8 version of the miner working without issue.  I have updated the repo.

**Comment by luteknarozniak** [comment]
Thank you! Just run a quick test - 1080Ti about 1.5-2.0MH/s increase; 1070 - about 1MH/s better than what I was getting in ccminer-tpruvot-2.2.5. Haven't played with intensity parameter yet.

**Comment by greerso** [comment]
Glad to hear that it worked out for you.  Let me know if there are other miners or features, that you would like to see added.  I can be found in the VoskCoin Discord if you want to chat live.

**Comment by luteknarozniak** [comment]
Yeah, it's been working great now. Thanks again.
Was very excited when I saw you have added enemy-1.08. 
Unfortunately it's not working for me. I'm getting "error while loading share libraries":
![image](https://user-images.githubusercontent.com/30240611/39090869-c8c15d92-45a6-11e8-9233-a456f190e84f.png)
Tried to replace few different miners - same result. Can't correctly swap them with enemy-1.08.
